api/envoy/config/filter/http/aws_lambda/v2/aws_lambda.proto

```diff
@@ -87,4 +87,10 @@ message AWSLambdaConfig {
   // `x-envoy-original-method` headers on the request to AWS lambda. 
   // Defaults to false.
   bool propagate_original_routing = 3;
+
+  // Sets cadence for refreshing credentials for Service Account.
+  // Does nothing if Service account is not set.
+  // Does not affect the default filewatch for service account only augments it.
+  // Defaults to not refreshing on time period. Suggested is 14 minutes.
```

Comment:
why 14 instead of something more round like 15 minutes?

Reply:
fwiw, k8s shared informer has similar model (event driven with polling as backup) and the polling is 10 minutes IIRC

Reply:
14 pulled from existing aws methodology as it polls frequently enough to work with minimal 1hour sessions and their 15 minute overlap

Reply:
I updated to 15 as I dont have strong feelings on this.